app/views/fragments/amiOverlay.scala.html

```diff
@@ -67,6 +67,13 @@
             </div>
         }
         <a href="#!" class="black-text modal-action modal-close waves-effect waves-white btn grey lighten-3">Close</a>
+        @if(ami.upgrade.isDefined) {
+            @stackStageApp.map { ssa =>
+                @ssa.riffRaffLink.map { link =>
+                    <a href="@link" target="_blank" rel="noopener" class="waves-effect waves-light btn">Redeploy now</a>
```

Comment:
I think just `Redeploy` would be clearer. `Redeploy now` makes it sound more like that button actually kicks off a deployment and is more verbose than required.